### ISSUE TITLE
A question about paper reading and course learning

### ISSUE THREAD

**Issue by sylvain-wei** [floriankark/cs224n-win2223#4]
I am surprised by your learning schedule because you can watch 2 courses and finish 2 papers' reading, and even you can finish one assignment on the same day! I just wonder if you have any tricks about paper reading or just you really go throught all the things... I have followed your schedule, and reading a paper on CBOW & Skip-gram model has cost me about half of the day... So I am very courious about how you can learn this course so fast... 🙇🏼 👍🏼

**Comment by floriankark** [comment]
Hi Sylvain! 

Thanks for reaching out and showing interest in my repo. I'm glad to share some insights with you.

Before diving into the course material, I had some prior knowledge in NLP, albeit at an introductory level, gained through personal projects and online resources. Additionally, I have a solid foundation in mathematics and Python/PyTorch programming, which helped me grasp concepts and work through the assignments more efficiently.

Here's a breakdown of my learning process:

1. **Preparation:** I started by reviewing the course syllabus and assignments. I took notes on topics that were new to me or sparked ideas based on my existing knowledge.
2. **Paper Reading:** I followed a quick skim-through approach when reading papers, focusing mainly on grasping the main ideas and results rather than diving deep into every detail. I found [this guide](https://web.archive.org/web/20231216162503/https://web.stanford.edu/class/ee384m/Handouts/HowtoReadPaper.pdf) helpful for structuring my reading. 
3. **Lecture Watching:** I watched the lectures at an accelerated speed (usually 1.5x to 2.0x) while referring to my notes. During the lectures, I clarified any unclear concepts and added more notes for better understanding.
4. **Assignment Tackling:** Armed with my notes, I approached the assignments. Whenever I encountered a concept I needed to revisit from the papers or lectures, I went back to that specific part for a deeper understanding.

It's worth mentioning that the initial assignments tend to be more manageable. However, as I progressed through the course, I realized the importance of revisiting papers for a deeper understanding. Taking more time to absorb the material is definitely not a sign of slow learning but rather a commitment to thorough comprehension. Feel free to adapt these strategies to your own pace and learning style! 🚀  And don't hesitate to ask more questions!

Cheers,
Florian
